assets/xml/objects/object_hunsui.xml

```diff
@@ -1,12 +1,12 @@
 ﻿<Root>
     <File Name="object_hunsui" Segment="6">
-        <DList Name="object_hunsui_DL_000220" Offset="0x220" />
+        <DList Name="object_hunsui_DL_000220" Offset="0x220" /> <!-- Original name is "m2_SEAhunsui_4_modelT" -->
         <Texture Name="object_hunsui_Tex_0003E0" OutName="tex_0003E0" Format="rgba16" Width="32" Height="32" Offset="0x3E0" />
         <TextureAnimation Name="object_hunsui_Matanimheader_000BF0" Offset="0xBF0" />
-        <Collision Name="object_hunsui_Colheader_000C74" Offset="0xC74" />
-        <DList Name="object_hunsui_DL_000EC0" Offset="0xEC0" />
+        <Collision Name="object_hunsui_Colheader_000C74" Offset="0xC74" /> <!-- Original name is "m2_SEAhunsui_4_bgdatainfo" -->
+        <DList Name="object_hunsui_DL_000EC0" Offset="0xEC0" /> <!-- Original name is "m2_SEAhunsui20_5_modelT" -->
```

Comment:
Maybe it's worth just softening the language a little due to the uncertainty? So `Original name is likely ""` or `Original name is inferred to be""`? I think it's still worth including in some capacity

Reply:
Yeah, that's totally fair. I'm definitely fairly certain about this, but you're totally right in that these *could* be swapped with each other, or just named something differently. Let me switch to something a little softer.

Reply:
Oh, you meant on the "20" thing! Whoops, I completely misunderstood what you meant. I thought you just meant the collision (because they are duplicates even in MM3D). Yeah, that's all fair too. Let me just put that for all the names on this object